### PR TITLE
[FEAT]Easter egg

### PR DESCRIPTION
**Pull request by rickfurt**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

This PR is a working in progress for the Easter holiday. 

related to the issue #542 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

# Checklist:

- [x] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [ ] Screenshot if it includes any UI changes
- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by rickfurt** [comment]
Hey @adamhannigan follow WIP as requested.

**Comment by danuribeiv** [comment]
Hey @rickfurt @adamhannigan @Rofrtd, could we use the egg images below? They fit more with the style of the game with the dark outlines. Also, the Easter Bunny NFT uses the same style from these eggs.

red egg orange egg yellow egg green egg blue egg purple egg pink egg

**Comment by danuribeiv** [comment]
Also, updated the shading of the Easter Basket: easter basket

**Comment by Rofrtd** [comment]
> ding of the

**Comment by Rofrtd** [comment]
> Also, updated the shading of the Easter Basket: easter basket

We sure can! ;)

File: src/features/game/events/collectEgg.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    2: 'Orange Egg',
+    3: 'Green Egg',
+    4: 'Yellow Egg',
+    5: 'Pink Egg',
+    6: 'Purple Egg'
+}
+
+const HUNT_START_AT_MS = Date.UTC(2022, 3, 16, 12)
+
+export function availableEgg(now: number = Date.now()): EasterEgg {
```
Comment (j923sneaks) [inline src/features/game/events/collectEgg.ts:16]: i know this is still WIP but shouldn't `now` (and `createdAt`) also be in UTC?

**Pull request merged**